doc/whats_new.rst

```diff
@@ -4,10 +4,18 @@
 NEW
 ---
 
+- :func:`nilearn.signal.clean` accepts new parameter `sample_mask`.
+  shape: (number of scans - number of volumes removed, )
+  Masks the niimgs along time/fourth dimension to perform scrubbing (remove
+  volumes with high motion) and/or non-steady-state volumes. Masking is applied
+  before signal cleaning.
+- All inherent classes of :class:`nilearn.input_data.BaseMasker` can use
```

Comment:
The reference to `BaseMasker` doesn't work since it is not exposed. CircleCI is failing because of this.